redisinsight/ui/src/pages/home/components/styles.module.ts

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+
+export const FixedWrapper = styled.div`
```

Comment:
Tip: You can use `FlexGroup` or `Col` instead of a regular `div`, so you don't have to define the flex properties on your own.

PS: The same note applies for the other classes bellow, as well.